### DISCUSSION TITLE
Which IDE do you use to develop Theia?

### DISCUSSION THREAD

**Discussion by hklene** [eclipse-theia/theia#14384] [General] [answered]
Hi

The second folder in the code repository is [`.vscode`](https://github.com/eclipse-theia/theia-ide/tree/master/.vscode). My naive assumption would be, you are all running Theia, to develop the next version of Theia?

Apart from the obvious PR-fail :D is there any other reason to stick to VSCode? Is there some feature you miss in Theia? Is there a roadmap of when Theia is ready to take over? Wasn't this the version 1.0? Or is that folder just a stale left over, that will soon be wiped out? Or maybe there is a good reason to keep it around?

*grin*
Holger

PS: Hope you can smile about it as well :D

**Comment by JonasHelming** [comment, marked as answer]
Hi,

many Theia developers use the Theia IDE as their development tool. We also do a preview release of the Theia IDE, which we test just before the official release. This "eat your own dogfood" pattern serves us very well for stabilization.
However we do neither track what people are using (except the mentioned testing phase), nor do we want to force anybody to use any particular tool. The .vscode folder contains artifacts also used by Theia, but you can also develop on the Theia project using VS Code or any other tool you want. 
I hope this clarifies you question :-)

Best regards,
Jonas
